starter_scripts/exaslct_within_docker_container_with_container_build.sh

```diff
@@ -2,7 +2,15 @@
 
 set -euo pipefail
 
-SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+[[ "$(uname)" = Darwin ]] && rl=greadlink || rl=readlink
```

Comment:
Not sure, if I like this construct. It is shorter than an if, but I feel it has more potential for errors.